pkgs/applications/audio/ams/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchgit
+, automake
+, alsaLib
+, ladspaH
+, libjack2
+, fftw
+, zita-alsa-pcmi
+, qt5
+, pkg-config
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "ams";
+  version = "unstable-2019-04-27";
+
+  src = fetchgit {
+    url = https://git.code.sf.net/p/alsamodular/ams.git;
```

Comment:
```suggestion
    url = "https://git.code.sf.net/p/alsamodular/ams.git";
```

Reply:
I misread this. I thought @jonringer was arguing for removing the quotes. Pardon my confusiong.